charts/osm/grafana/dashboards/osm-control-plane.json

```diff
@@ -60,7 +60,10 @@
         }
       ]
     },
-    "description": "OSM Control Plane Metrics latest",
+    "title": "OSM Control Plane Metrics",
+    "description": "Compatible with OSM v1.0.0. This dashboard provides traffic metrics from the given service to OSM’s control plane.",
```

Comment:
my concert with specifying the version here adds an overhead of updating this every time there's a new release

Reply:
Could the version be an input variable like DS_PROMETHEUS or CONTROL_PLANE_NAMESPACE?

Reply:
I misspoke in my previous comment. I am in favor of updating the **description** to include the OSM version.

Also, an input variable would not make sense for the OSM version. Perhaps something like with would work https://grafana.com/docs/grafana/latest/administration/configuration/#env-provider